action.yaml

```diff
@@ -54,6 +57,11 @@ runs:
         echo "::set-output name=cache-key::$result"
         echo "::set-output name=cache-key-suffix::$suffix"
       shell: bash
+    - name: Configure git client with github auth token
```

Comment:
Nitpicks:
- I prefer the `name` to be written in a heading style, e.g. "Configure Git Client with GitHub Auth Token", since the `name` basically becomes a section within the GHA output.
- An `id` is only required if referring to the step, such as when accessing the output of the step. Here, the `id` does not really add anything.

Reply:
PR updated.